settings.gradle.kts

```diff
@@ -60,5 +61,6 @@ project(":appyx-components:backstack:common").name = "backstack"
 project(":appyx-components:spotlight:common").name = "spotlight"
 project(":appyx-components:demos:common").name = "demos"
 project(":appyx-interactions:common").name = "appyx-interactions"
+project(":appyx-components:demos:web").name = "puzzle-15-web"
```

Comment:
This component isn't necessarily representative of all of the appyx-components demos. Let's move it to its own module inside demos.

Reply:
i renamed the module to `demos-web` instead, it seems too much to have a dedicated module for this game. This now sits at the same level with Promoter and Cards (see appyx-components:demos:common -> commonMain)